static/js/dashboard_routes.js

```diff
@@ -37,7 +38,9 @@ export function makeDashboardRoutes(browserHistory, store, onRouteUpdate) {
               <Route path="privacy" component={PrivacyTab} />
             </Route>
             <Route path="/terms_of_service" component={TermsOfServicePage} />
-            <Route path="/users/:username" component={UserPage} />
+            <Route path="/users" component={UserPage} >
```

Comment:
Is there any case where we would want to use `UserPage` without `User`? When the user goes to `/users` they will see `UserPage` without any child components